package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "css-frameworks",
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "build": "sass src/assets/scss:dist/css",
+    "watch": "sass --watch src/assets/scss:dist/css & live-server"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/trine-marie-sandberg/css-frameworks.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/trine-marie-sandberg/css-frameworks/issues"
+  },
+  "homepage": "https://github.com/trine-marie-sandberg/css-frameworks#readme",
+  "dependencies": {
+    "bootstrap": "^5.2.2"
+  },
+  "devDependencies": {
+    "sass": "^1.55.0"
```

Comment:
Or install it to here with `npm i -D live-server`

Reply:
I will consider this. Thank you :)